content/blog/2021/01/camel-quarkus-release-1.6.0/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Camel Quarkus 1.6.0 Released"
+date: 2021-01-18
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 1.6.0 Released"
+summary: "The highlights of Camel Quarkus 1.6.0"
+---
+
+<sub><sup>Original image by <a href="https://commons.wikimedia.org/wiki/User:99of9">Toby Hudson</a> <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a> via <a href="https://en.wikipedia.org/wiki/Camel_racing#/media/File:CamelRacingCamelCup2009Heat.JPG">Wikipedia</a></sup></sub>
+
+We are pleased to announce the release 1.6.0 of Camel Quarkus! Here are the highlights.
+
+## Java 11 required
+
+Having deprecated Java 8 [long time ago](https://camel.apache.org/blog/2020/04/Camel-Quarkus-release-1.0.0-M7/) and
+following the same requirement by Quarkus, Camel Quarkus now requires Java 11+.
+
+## Support for more Camel components
+
+Camel Quarkus brings support for three new Camel components:
+
+* [JSONATA](https://camel.apache.org/camel-quarkus/latest/reference/extensions/jsonata.html)
+* [Vert.x Kafka](https://camel.apache.org/camel-quarkus/latest/reference/extensions/vertx-kafka.html)
+* [Redis](https://camel.apache.org/camel-quarkus/latest/reference/extensions/redis.html) (JVM only)
+
+Components newly supported in native mode:
+
+* [Azure Storage Blob Service](https://camel.apache.org/camel-quarkus/latest/reference/extensions/azure-storage-blob.html)
+* [Debezium MongoDB Connector](https://camel.apache.org/camel-quarkus/latest/reference/extensions/debezium-mongodb.html)
+* [Google BigQuery](https://camel.apache.org/camel-quarkus/latest/reference/extensions/google-bigquery.html)
```

Comment:
The google big query page indicates nativeSince = 1.5.0. Maybe we need to remove from release note ?

Reply:
Ah ok, the jvmSince was not correct. It is fixed in this [commit](https://github.com/apache/camel-quarkus/commit/7d8d55f44ccb79fe688d3d2e13e818c6756035cf).

Reply:
Yes, exactly.